release/release.md

```diff
@@ -0,0 +1,122 @@
+# How to do a Kata Containers Release
+
+* [Introduction](#Introduction)
+* [Requirements](#Requirements)
+* [Release process](#release-process)
+
+# Introduction
+
+If you are reading this document, you might want to create a Kata Containers
+Release.
+
+The Kata Containers Release Process is defined in the following
+[document][release-process-definition]. To simplify this process, we have
+created a [Release Checklist][release-checklist]. Finally, to simplify the
+Release Checklist process we have automated most of the process, this document
+guides on how to use release scripts instead of do all the checklist manually.
+
+
+## Requirements
+
+- It is recommended to have at least 2 GB of free disk space to perform these tasks.
+
+- [Go](https://golang.org/doc/install#)
```

Comment:
It might be worth linking to https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#requirements-to-build-individual-components to ensure the user installs the correct golang version?